### DISCUSSION TITLE
Como faço para fazer comando de ping em javascript

### DISCUSSION THREAD

**Discussion by s0v1etic** [s0v1etic/s0v1etic#3] [General]
Help

**Comment by guilherme4u** [comment]
const Discord = require("discord.js")

module.exports = {
    name: "ping",
    aliases: [""],

    run: async(client, message, args) => {

        let embed = new Discord.EmbedBuilder()
            .setColor("Random")
            .setAuthor({ name: client.user.username })
            .setDescription(`Olá ${message.author}, seu ping está em: \`carregando...\`.`);

            let embed2 = new Discord.EmbedBuilder()
            .setColor("Random")
            .setAuthor({ name: client.user.username })
            .setDescription(`Olá ${message.author}, seu ping está em: \`${client.ws.ping}\`.`);

            message.reply({embeds: [embed], }).then(msg => {
                setTimeout( () => {
                    msg.edit({ embeds: [embed2] })
                }, 3000)
            })
    } 


etste é um bom codigo para ping em js

**Reply by s0v1etic** [reply]
obrigado.
